### DISCUSSION TITLE
Running Ventura OS from Monterey host

### DISCUSSION THREAD

**Discussion by pavellety** [cirruslabs/tart#523] [Q&A] [answered]
I would like to run OSX Ventura on my Monterey host but tart is telling me that the host version is outdated. Is this expected (Based on official doc it should be ok)? Do I need to have same version for host/VM. I am using tart 1.6.0

```
> ~ % tart clone ghcr.io/cirruslabs/macos-ventura-base:latest ventura-test
> ~ % tart run --no-graphics ventura-test
Error: error: host macOS version is outdated to run this virtual machine
```

Thanks and Regards,
Pavel

**Comment by edigaryev** [comment]
Looks like a duplicate of https://github.com/cirruslabs/tart/issues/469.

You either need to upgrade your host to Ventura, or run Monterey guests.

**Comment by fkorotkov** [comment, marked as answer]
You can only run Ventura version 13.2.1 and lower on Monterey. To run Ventura 13.3 which the latest base image is based on requires the host to be also Ventura.

Please try `ghcr.io/cirruslabs/macos-ventura-vanilla:13.2.1`

**Reply by pavellety** [reply]
Thanks, is this written somewhere? I guess this could be interesting information for others as well.
